components/MeetupsPage/guidlines.js

```diff
@@ -0,0 +1,198 @@
+              <div className="bg-blue-500 rounded-full p-1 mr-3">
+                <svg xmlns="http://www.w3.org/2000/svg" className="h-5 w-5 text-white" viewBox="0 0 20 20" fill="currentColor">
+                  <path fillRule="evenodd" d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7-4a1 1 0 11-2 0 1 1 0 012 0zM9 9a1 1 0 000 2v3a1 1 0 001 1h1a1 1 0 100-2h-1v-3a1 1 0 00-1-1z" clipRule="evenodd" />
+                </svg>
+              </div>
+              <h2 className="text-white text-xl font-bold">Guidelines</h2>
+            </div>
+            
+            {/* Guidelines List */}
+            <div className="space-y-6">
+              <div className="flex">
+                <div className="bg-green-500 rounded-full p-1 mr-3 h-6 w-6 flex-shrink-0 mt-1">
+                  <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 text-white" viewBox="0 0 20 20" fill="currentColor">
+                    <path fillRule="evenodd" d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z" clipRule="evenodd" />
+                  </svg>
+                </div>
+                <div>
+                  <h3 className="text-white font-medium">Company Sponsorship</h3>
+                  <p className="text-gray-300 text-sm mt-1">Companies can sponsor by hosting meetups or providing equipment for streaming/recording.</p>
+                </div>
+              </div>
+              
+              <div className="flex">
+                <div className="bg-green-500 rounded-full p-1 mr-3 h-6 w-6 flex-shrink-0 mt-1">
+                  <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 text-white" viewBox="0 0 20 20" fill="currentColor">
+                    <path fillRule="evenodd" d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z" clipRule="evenodd" />
+                  </svg>
+                </div>
+                <div>
+                  <h3 className="text-white font-medium">Content Focus</h3>
+                  <p className="text-gray-300 text-sm mt-1">Meetups should be educational and focus on knowledge sharing rather than sales or marketing.</p>
+                </div>
+              </div>
+              
+              <div className="flex">
+                <div className="bg-red-500 rounded-full p-1 mr-3 h-6 w-6 flex-shrink-0 mt-1">
+                  <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 text-white" viewBox="0 0 20 20" fill="currentColor">
+                    <path fillRule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zM8.707 7.293a1 1 0 00-1.414 1.414L8.586 10l-1.293 1.293a1 1 0 101.414 1.414L10 11.414l1.293 1.293a1 1 0 001.414-1.414L11.414 10l1.293-1.293a1 1 0 00-1.414-1.414L10 8.586 8.707 7.293z" clipRule="evenodd" />
+                  </svg>
+                </div>
+                <div>
+                  <h3 className="text-white font-medium">No Sales Talks</h3>
```

Comment:
I think it should rather be a requirement, no?

Reply:
Done!